File: telerik_ui_for_winforms_courseware_chm/Courseware/Treeview/VB/DatabindingTreeView/Databinding/XMLBinding.vb
Public Class XMLBinding

    Private Sub XMLBinding_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Dim dataSet As DataSet = New DataSet()
        dataSet.ReadXml("toc.xml")
        Me.radTreeView1.DataMember = "item"
        Me.radTreeView1.DisplayMember = "title"
        Me.radTreeView1.ChildMember = "id"
        Me.radTreeView1.ParentMember = "parentId"
        Me.radTreeView1.DataSource = dataSet
    End Sub
End Class

File: telerik_ui_for_winforms_courseware_chm/Courseware/Treeview/CS/DatabindingTreeView/Databinding/XMLBinding.cs
using System;
using System.Data;
using Telerik.WinControls.UI;

namespace Databinding
{
    public partial class XMLBinding : RadForm
    {
        public XMLBinding()
        {
            InitializeComponent();
        }

        private void XMLBinding_Load(object sender, EventArgs e)
        {
            DataSet dataSet = new DataSet();
            dataSet.ReadXml("toc.xml");
            this.radTreeView1.DataMember = "item";
            this.radTreeView1.DisplayMember = "title";
            this.radTreeView1.ChildMember = "id";
            this.radTreeView1.ParentMember = "parentId";
            this.radTreeView1.DataSource = dataSet;
        }
    }
}
